src/structs.jl

```diff
@@ -3,7 +3,7 @@ import StructTypes: StructType, DictType, ArrayType, StringType, NumberType, Boo
 read(io::IO, ::Type{T}; kw...) where {T} = read(Base.read(io, String), T; kw...)
 read(bytes::AbstractVector{UInt8}, ::Type{T}; kw...) where {T} = read(VectorString(bytes), T; kw...)
 
-function read(str::AbstractString, ::Type{T}; kw...) where {T}
+function _buf_pos_len_b(str::AbstractString, ::Type{T}) where {T}
```

Comment:
I'd rather call this `initial` or `prepare`; `_buf_pos_len_b` is pretty obscure

Reply:
Done.